docs/documentation/indexing/create_index.mdx

```diff
@@ -38,25 +38,26 @@ WITH (key_field='<key_field>');
   record. Usually, this is the same column that is the primary key of the table.
 </ParamField>
 
-## Indexing Settings
+## Optimizing Index Build Times
 
-Several [settings](/documentation/configuration/index) exist to control the amount of resources to allocate during index creation.  While they have sensible
-defaults, overall indexing performance can be drastically improved if configured for the host system.
+By default, Postgres' `maintenance_work_mem` allows `64MB` of indexing memory. For large indexes, a higher `maintenance_work_mem`
+can speed up index build times significantly.
+
+```sql
+-- Should not exceed the server's RAM
+SET maintenance_work_mem = '16GB';
+```
+
+You can read more about indexing settings [here](/documentation/configuration/index).
 
 ## Choosing a Key Field
 
+The `key_field` must be a **unique** column in the table. A non-unique key field is likely to lead to undefined behavior
```

Comment:
Might be more clear to say that the `key_field` should have a `UNIQUE` constraint.